modules/extension/wms/src/main/java/org/geotools/ows/wms/map/WMSLayer.java

```diff
@@ -88,6 +88,18 @@ public WMSLayer(WebMapServer wms, Layer layer, String style) {
         super(new WMSCoverageReader(wms, layer, style), STYLE);
     }
 
+    /**
+     * Builds a new WMS layer
+     *
+     * @param wms
+     * @param layer
+     * @param style
+     * @param format
+     */
```

Comment:
Yes that's what a constructor usually do, this Javadoc doesn't add any valuable information.

Reply:
will remove it